package.json

```diff
@@ -63,7 +63,7 @@
     "react-sticky": "^6.0.3",
     "react-testing-library": "^6.0.0",
     "smooth-scroll": "^16.1.2",
-    "styled-components": "^4.1.3",
+    "styled-components": "^5.2.1",
```

Comment:
Lens had a `peerDependency` of `styled-components >= 5.2.1`.

Instead of lowering it there, I bumped it up here since AFAIK there aren't any breaking changes. If this was a conscious decision though, let me know, I'll revert this and lower the `peerDependency` in Lens